templates/mips/qemu/build.conf

```diff
@@ -3,8 +3,12 @@ TARGET = embox
 
 ARCH = mips
 
-CROSS_COMPILE = mips-elf-
+CROSS_COMPILE = mips-linux-gnu-
```

Comment:
Our "official" crosstool is mips-elf, so this template became broken out-of-box.
bare-metal toolchain should be mips-elf or modern mips-none-... mips-linux-gnu creates linux mips binaries, not bare-metal (standalone, freestanding)